executioncontext/projectview/src/test/java/org/jetbrains/bsp/bazel/projectview/model/ProjectViewBuilderTest.kt

```diff
@@ -493,7 +503,8 @@ class ProjectViewBuilderTest {
                             ),
                             buildFlags = ProjectViewBuildFlagsSection(
                                     List.of("--build_flag4.1=value4.1", "--build_flag4.2=value4.2")
-                            )
+                            ),
+                            buildManualTargets = null,
```

Comment:
u can remove it

Reply:
I made them null for now to see that all tests passed like that, now I am trying to change them